### ISSUE TITLE
Latest python-django-mptt 0.7.1 breaks tests

### ISSUE THREAD

**Issue by brianmay** [Karaage-Cluster/karaage#263]
```
ERRORS:
karaage.Project.level: (models.E006) The field 'level' clashes with the field 'level' from model 'karaage.project'.
karaage.Project.lft: (models.E006) The field 'lft' clashes with the field 'lft' from model 'karaage.project'.
karaage.Project.rght: (models.E006) The field 'rght' clashes with the field 'rght' from model 'karaage.project'.
karaage.Project.tree_id: (models.E006) The field 'tree_id' clashes with the field 'tree_id' from model 'karaage.project'.
```

**Comment by brianmay** [comment]
Reported against django-mptt at https://github.com/django-mptt/django-mptt/issues/375

**Comment by brianmay** [comment]
This should be fixed in django-mptt 0.7.3, need to test.

**Comment by brianmay** [comment]
I believe this is fixed. Closing.
